### ISSUE TITLE
go get subdirectories

### ISSUE THREAD

**Issue by bobheadxi** [bobheadxi/twist#3]
[/u/earthboundkid](https://www.reddit.com/user/earthboundkid) pointed out this post:

https://npf.io/2016/10/vanity-imports-with-hugo/

It mentions a workaround for using `go get` on nested directories, which Twist doesn't account for right now:

```
❯ go get go.bobheadxi.dev/gobenchdata/x/gobenchdata-web
package go.bobheadxi.dev/gobenchdata/x/gobenchdata-web: unrecognized import path "go.bobheadxi.dev/gobenchdata/x/gobenchdata-web" (parse https://go.bobheadxi.dev/gobenchdata/x/gobenchdata-web?go-get=1: no go-import meta tags ())
```

**Comment by bobheadxi** [comment]
fixed in 2ab6e92

**Comment by bobheadxi** [comment]
wait a second...
